### ISSUE TITLE
`gradle-intellij-plugin` v1.13.0 causes ` Unsupported JVM architecture was selected for running Gradle tasks: x86_64. Supported architectures: amd64, aarch64`

### ISSUE THREAD

**Issue by PawelLipski** [VirtusLab/git-machete-intellij-plugin#1536]
Nothing suspicious in https://github.com/JetBrains/gradle-intellij-plugin/releases/tag/v1.13.0.

```
> Task :backend:impl:test FAILED

> Task :frontend:actions:compileJava
Note: /Users/pawel_lipski/git-machete-intellij-plugin/frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/toolbar/OpenMacheteFileAction.java uses or overrides a deprecated API.
Note: Recompile with -Xlint:deprecation for details.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':backend:impl:test'.
> Unsupported JVM architecture was selected for running Gradle tasks: x86_64. Supported architectures: amd64, aarch64
```

Probably happening where Gradle first needs to fork off the JVM (?) in a subproject where the plugin is enabled. It's not always happening on `:backend:impl:test`.

Pinned the version in the catalog to v1.12.0. Let's wait for another gradle-intellij-plugin release, maybe it's gonna get itself sorted out.

**Comment by PawelLipski** [comment]
https://github.com/JetBrains/gradle-intellij-plugin/issues/1317

**Comment by PawelLipski** [comment]
Will be fixed with 1.13.1, see https://github.com/JetBrains/gradle-intellij-plugin/issues/1317#issuecomment-1434831145
